README.md

```diff
@@ -137,6 +137,35 @@ COVID-19/
 
 *Viene messo a disposizione un file JSON complessivo di tutte le date nella cartella "dati-json": dpc-covid19-ita-andamento-nazionale.json*
 
+### API
+
+Tutte le chiamate accettano i parametri `begin` e `end` in formato ISO 8601.
```

Comment:
Puoi fare una specifica OpenAPI anziché indicarle nel readme, eg: https://editor.swagger.io/?url=https%3A%2F%2Fraw.githubusercontent.com%2Fteamdigitale%2Fapi-starter-kit%2Fmaster%2Fopenapi%2Fsimple.yaml

I can create a template for you.

This code could be interesting for creating an api-starter-kit-rust project like https://github.com/teamdigitale/api-starter-kit-python. WDYT

Reply:
> Puoi fare una specifica OpenAPI anziché indicarle nel readme, eg: https://editor.swagger.io/?url=https%3A%2F%2Fraw.githubusercontent.com%2Fteamdigitale%2Fapi-starter-kit%2Fmaster%2Fopenapi%2Fsimple.yaml

Sure. I think we should keep them in the README for the time being though, at least until we have a place to serve the rendered version.

I'll wait for the new format coming out today (see https://github.com/pcm-dpc/COVID-19/issues/115#issuecomment-597799832)

> This code could be interesting for creating an api-starter-kit-rust project like https://github.com/teamdigitale/api-starter-kit-python. WDYT

That's a great idea, but be aware: as other Rustaceans may have noticed, my Rust is pretty basic.

Reply:
@ioggstream WTH, OpenAPI [doesn't support optional path parameters yet](https://github.com/OAI/OpenAPI-Specification/issues/93). The documentation won't be 100% accurate.

Reply:
Looks like the format haven't change yet, so I added the spec anyway.